relayer-cli/tests/acceptance.rs

```diff
@@ -45,3 +45,15 @@ fn start_no_args() {
     ); // Todo: find out how to disable colored output and then remove the `[^ ]*` part from the regexp.
     cmd.wait().unwrap().expect_success();
 }
+
+#[cfg(not(tarpaulin))]
+#[test]
+fn example_configuration_is_valid() {
```

Comment:
Nit: There's another file `relayer/tests/config/fixtures/relayer_conf_example.toml` which is used for a similar test in `parse_valid_config()` -> `relayer/src/config.rs:448`. Thinking if we should remove that file and test?

Reply:
+1

Reply:
I think both are useful: we are interested in the example config to not bit rot, and relayer lib's test probably needs its conf test vector close by.

Reply:
There's also a difference in software under test between the two tests: the test here runs the executable, the test in the lib crate is only a unit test.